src/AppServices/Validations/CustomerValidators.cs

```diff
@@ -1,11 +1,10 @@
+using DomainModels;
 using FluentValidation;
-using DesafioWarren.WebAPI.Models;
 
-namespace DesafioWarren.WebAPI.Validations;
-
-public class CustomerValidator : AbstractValidator<Customer>
+namespace AppServices;
+public class CustomerValidators : AbstractValidator<Customer>
```

Comment:
Não entendi o plural, está validando mais de 1 entidade?
```suggestion
public class CustomerValidator : AbstractValidator<Customer>
```